marketplace/createUIDefinition.json

```diff
@@ -0,0 +1,229 @@
+							"value": false
+						}
+					],
+					"required": true
+				},
+				"visible": true
+			}
+		],
+		"steps": [
+			{
+				"name": "clusterDetails",
+				"label": "Cluster Details",
+				"elements": [
+					{
+						"name": "existingClusterSection",
+						"type": "Microsoft.Common.Section",
+						"elements": [
+							{
+								"name": "clusterLookupControl",
+								"type": "Microsoft.Solutions.ArmApiControl",
+								"request": {
+									"method": "GET",
+									"path": "[concat(subscription().id, '/resourcegroups/', resourceGroup().name,  '/providers/Microsoft.ContainerService/managedClusters?api-version=2022-03-01')]"
+								}
+							},
+							{
+								"name": "existingClusterResourceName",
+								"type": "Microsoft.Common.DropDown",
+								"label": "AKS Cluster Name",
+								"toolTip": "AKS Cluster Resource Name",
+								"constraints": {
+									"allowedValues": "[map(steps('clusterDetails').existingClusterSection.clusterLookupControl.value, (item) => parse(concat('{\"label\":\"', item.name, '\",\"value\":\"', item.name, '\"}')))]",
+									"required": true
+								}
+							}
+						],
+						"visible": "[equals(basics('createNewCluster'), false)]"
+					},
+					{
+						"name": "newClusterSection",
+						"type": "Microsoft.Common.Section",
+						"elements": [
+							{
+								"name": "aksVersionLookupControl",
+								"type": "Microsoft.Solutions.ArmApiControl",
+								"request": {
+									"method": "GET",
+									"path": "[concat(subscription().id, '/providers/Microsoft.ContainerService/locations/', location(),  '/orchestrators?api-version=2019-04-01&resource-type=managedClusters')]"
+								}
+							},
+							{
+								"name": "newClusterResourceName",
+								"type": "Microsoft.Common.TextBox",
+								"label": "AKS cluster name",
+								"defaultValue": "",
+								"toolTip": "Use only allowed characters",
+								"constraints": {
+									"required": true,
+									"regex": "^[a-z0-9A-Z]{6,30}$",
+									"validationMessage": "Only alphanumeric characters are allowed, and the value must be 6-30 characters long."
+								}
+							},
+							{
+								"name": "kubernetesVersion",
+								"type": "Microsoft.Common.DropDown",
+								"label": "Kubernetes version",
+								"toolTip": "The version of Kubernetes that should be used for this cluster. You will be able to upgrade this version after creating the cluster.",
+								"constraints": {
+									"allowedValues": "[map(steps('clusterDetails').newClusterSection.aksVersionLookupControl.properties.orchestrators, (item) => parse(concat('{\"label\":\"', item.orchestratorVersion, '\",\"value\":\"', item.orchestratorVersion, '\"}')))]",
+									"required": true
+								}
+							},
+							{
+								"name": "vmSize",
+								"type": "Microsoft.Compute.SizeSelector",
+								"label": "VM size",
+								"toolTip": "The size of virtual machine for VM.",
+								"recommendedSizes": [
+                  "Standard_D2ds_v5",
```

Comment:
**question:** Any specific reasons for these SKUs?

Reply:
Good question: no.  These just happened to be the ones we've been running performance tests on :)  I didn't quite know what to supply here.  Maybe we just leave blank?  Or, happy to update with different/additional SKUs.

Reply:
Not yet; not that I know of.  More a case of lowered performance on less powerful SKUs, as one would expect.

Okay, will leave blank.